pydash/numerical.py

```diff
@@ -483,3 +523,6 @@ def rounder_func(item):
             pass
 
     return result
+
+
+subtract = partial(call_math_operator, callback=operator.sub)
```

Comment:
Le'ts not use a partial here. Can just call `call_math_operator` inside the `subtract` function.